docs/release-notes/release-notes-0.18.0.md

```diff
@@ -80,6 +80,11 @@
   the new payment status `Payment_INITIATED` should be used for payment-related
   RPCs. It's recommended to use it to provide granular controls over payments.
 
+* When publishing transactions in `lnd`, all the transactions will now go
```

Comment:
additional detail here would be helpful for the users to understand how/what will this improve

Reply:
updated a bit, I don't think the users need to know beyond that now the txes will be checked by calling `testmempoolaccept`

Reply:
Yep, for users it's mainly: `lnd` won't publish stuff that it knows won't be accepted to the mempool.